src/devices/Camera/tests/TestCamera/ProcessRunnerTests.cs

```diff
@@ -0,0 +1,159 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Text;
+
+using Iot.Device.Camera;
+using Iot.Device.Camera.Settings;
+
+namespace TestCamera;
+
+/// <summary>
+/// Unit tests for the Camera binding
+/// </summary>
+public class ProcessRunnerTests
+{
+    private const string FakeVideocapture =
+        @"..\..\..\..\FakeVideoCapture\bin\Debug\net6.0\FakeVideoCapture.exe";
```

Comment:
I am not sure this is the best way in our repo to reference the external process as this string contains the `net6.0` tfm.

An alternative would be to reference the project from the unit tests so that the exe is copied in its binary folder.

Other ideas?

Reply:
Yes, you can for example do the following:

* search for this file in a recursive way up to the moment you'll find one. 
* Or, you can as well copy it into your test project. You can then set it to copy in the artifact. You just need to build the FakeVideoCapture project first.

Reply:
Thank you @Ellerbach 
- I was going with the option 1 but there no Initialization phase in xunit
- Not sure how to do the option 2 (both phases)

But I tried to simply add a reference to the exe and it gets copied to the unit test project binary folder.
Do you see problems with this solution?

Reply:
BTW I just committed this solution as it looks good.